system/core/document.php

```diff
@@ -514,6 +514,28 @@ public function findField($name, $tagName) {
 			return $found->first();
 		}
 
+		/**
+		 * Add an option element to the given field (as returned
+		 * by findField), using the given parameters.
+		 *
+		 * @param HyphaDomElement $field
+		 * @param string $value
+		 * @param string $content
+		 * @param array $attributes
+		 *
+		 * @return HyphaDomElement The created option element
+		 */
+		public function addOptionToSelect($field, $value, $content, $attributes = []) {
```

Comment:
use type hinting at `$field`.

Reply:
and also at `$attributes`

Reply:
Right, I did the comments, but then it is still useful to use PHP-readable hints as well, of course.

Fixed.